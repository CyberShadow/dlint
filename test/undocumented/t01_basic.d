void bad01() {}

private void good01() {}

private
{
	void good02() {}
}

/// documented!
void good03() {}

void good04() {} /// ditto

struct bad1
{
	void bad11() {}
private:
	void good12() {}
}

private struct good2
{
	void good21() {}
public:
	void good22() {}
}

void bad3()
{
	int good31;
}

struct bad4(T)
{
	void bad41() {}
private:
	void good42() {}
}

