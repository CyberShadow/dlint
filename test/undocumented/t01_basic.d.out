undocumented/t01_basic.d(1): Undocumented public declaration: FuncDeclaration `bad01`
undocumented/t01_basic.d(15): Undocumented public declaration: StructDeclaration `bad1`
undocumented/t01_basic.d(17): Undocumented public declaration: FuncDeclaration `bad11`
undocumented/t01_basic.d(29): Undocumented public declaration: FuncDeclaration `bad3`
undocumented/t01_basic.d(34): Undocumented public declaration: TemplateDeclaration `bad4(T)`
undocumented/t01_basic.d(36): Undocumented public declaration: FuncDeclaration `bad41`
